roles/delius-core/templates/NDelius.properties.j2

```diff
@@ -64,6 +64,8 @@ APTRACKER_ERRORS_URL={{ aptracker_api_errors_url | default('') }}
 
 GDPR_URL=/gdpr/ui/homepage
 
+PREPARE_CASE_FOR_SENTENCE_URL={{prepare_a_case_url}}
```

Comment:
We should add a default so that bootstrapping doesn't fail for environments where the config value isn't set:
```suggestion
PREPARE_CASE_FOR_SENTENCE_URL={{ prepare_a_case_url | default('') }}
```

.
Alternatively you can omit the line altogether if the config is missing:
```suggestion
{% if prepare_a_case_url is defined %}
PREPARE_CASE_FOR_SENTENCE_URL={{ prepare_a_case_url | default('') }}
{% endif %}
```